packages/cli/src/commands/sandbox/sandbox_command.ts

```diff
@@ -196,6 +219,30 @@ export class SandboxCommand
           boolean: true,
           global: false,
         })
+        .option('stream-function-logs', {
+          describe:
+            'Whether to stream function execution logs. Default: false. Use --function-names to filter for specific functions to stream logs from',
+          boolean: true,
+          global: false,
+          group: 'Logs streaming',
+        })
+        .option('logs-filter', {
+          describe: `Regex pattern to filter logs from only matched functions. E.g. to stream logs for a function, specify it's name, and to stream logs from all functions starting with auth specify 'auth' Default: Stream all logs`,
```

Comment:
Shouldn't a regex for "functions starting with auth" be `^auth.*`? Also, why are we supporting regex and array input? You can make a regex to support multiple disjoint strings with `(foo|bar|baz)`

IMO if we are going to support regex, it should just be a single arg

Reply:
1. For a full regex match yes, but I'm going for a partial string match as in my opinion it's easier to specify and also a default for pretty much all regex matchers. I don't see why we would prevent partial matching.
2. Similarly providing array just makes it easier to provide input instead of creating complex regex pattern. @josefaidt to provide more input here.

Reply:
:wave: yes the important piece of regex is the wildcard match (e.g. `auth*` picks up all functions named with `auth` like `auth-post-confirmation` and `auth-pre-signup`) but it is much easier to specify multiple patterns than it is to craft complex regex when writing the command

```
--logs-filter auth* --logs-filter resolver*
```

I think regex can be a bit of a hurdle though compared to a glob pattern

Reply:
Do we want to support glob patterns or regex then? The current impl is using regex which would require `--logs-filter auth.* logs-filter resolver.*`

Reply:
glob patterns don't make sense for non pathlike strings. Globs are typically (or only?) used for pathname expansions.

Reply:
There are use cases for glob patterns outside filenames
https://www.sqlitetutorial.net/sqlite-glob/
https://duckdb.org/docs/sql/functions/pattern_matching.html#glob

like [minimatch](https://github.com/isaacs/minimatch) without the globstar

Reply:
Minimatch is basically a glob implementation, still only relevant for pathlike string matching, not for any arbitrary matches.

In the SQL world, the `glob` is basically implemented as a regex, e.g. there is no such thing as `**`.